localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -1026,7 +1026,7 @@ geometry_msgs::msg::PoseWithCovarianceStamped NDTScanMatcher::align_pose(
   const double stddev_pitch = std::sqrt(covariance(4, 4));
 
   // Since only yaw is uniformly sampled, we define the mean and standard deviation for the others.
-  const std::vector<double> sample_mean{
+  std::vector<double> sample_mean{
```

Comment:
How about this variable?

Reply:
thank you again for mentioning out!
I fixed it at ae7b866 .